core/domain/platform_feature_services.py

```diff
@@ -321,3 +325,25 @@ def get_platform_parameter_value(
     context = _create_evaluation_context_for_server()
     param = registry.Registry.get_platform_parameter(parameter_name)
     return param.evaluate(context)
+
+
+def feature_flag_valid_on_current_server(feature_stage: str) -> bool:
+    """Checks if the feature is valid on the current server.
+
+    Args:
+        feature_stage: str. The stage of the feature flag.
+
+    Returns:
+        bool. Returns True if the feature is valid on current server else False.
```

Comment:
bool. Whether the feature is valid on the current server.

Reply:
No longer valid as the logic has been moved.